### ISSUE TITLE
[GH Request] I was erroneously granted maintainer access to cs_comments_service

### ISSUE THREAD

**Issue by regisb** [openedx/axim-engineering#727]
### Firm Name

Overhang.IO

### Urgency

Low (2 weeks)

### Requested Change

Remove my access rights from the [cs_comments_service](https://github.com/openedx/cs_comments_service) repository.

### Reasoning

On Friday April 14th at 17:28 (Paris time) I received a GitHub email:

>  [GitHub] Subscribed to openedx/cs_comments_service notifications 
> ...
>  You were automatically subscribed because you’ve been given access to the repository.

And sure enough, I am now able to merge and close pull requests from that repo:

![Screenshot from 2023-04-17 12-05-54](https://user-images.githubusercontent.com/44319/232453660-9ab5cb8f-18cd-42d1-a579-fc90e13d950f.png)

I am not a maintainer of that repo, so I was granted access by error. Please stop me before I do anything bad to that repo!

**Comment by sarina** [comment]
Good catch. `cs_comments_service` had been added with write access to https://github.com/orgs/openedx/teams/roadmap-editors/repositories - I've removed that access.
